secret-squirrel.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
Todo: either change `secret squirrel` or stop using `precise-commits`

Reply:
So Secret Squirrel generates a file that uses spaces and not tabs? That's different to the [next convention](https://github.com/Financial-Times/n-gage/blob/master/dotfiles/.editorconfig#L8) :scream_cat: 

Although weirdly it looks like Secret Squirrel is [trying to generate this file with tabs](https://github.com/Financial-Times/secret-squirrel/blob/master/lib/init.js#L34) :thinking:

Reply:
It's an odd one